src/renderer/views/History/History.js

```diff
@@ -19,10 +19,7 @@ function filterVideosWithQuery(videos, query, attrProcessor = identity) {
     } else if (typeof (video.author) === 'string' && attrProcessor(video.author).includes(query)) {
       return true
     }
-
     return false
-  }).sort((a, b) => {
```

Comment:
I haven't tested this yet and have not worked much in `history.js`, but from what I can tell, you are correct that this does seem to be something that should already be being handled by `history.js` and thus does not need any manual work being done here. If @absidue or @PikachuEXE have context on why this is being done here as well, we can add it back with a code comment containing that explanation, but this is fine from what I can see.

Reply:
It's from https://github.com/FreeTubeApp/FreeTube/commit/dbf69f242ad01976968e3527a024bc806c1d7974
Not sure if it's still needed

Reply:
Another sorting is introduced 3 months ago
https://github.com/FreeTubeApp/FreeTube/commit/01dbdaddfb71e6473f54316ae0dd1b82dd7419c3
I will let @absidue decide if this is still needed

Reply:
I think it should be fine to remove it here, as the history store loads it sorted from the database and keeps the array in the right order when updating the history. The sort that I added for the history import in the store a few months ago is because the import replaces the array (as it has already merged the histories at the point) so it needs to be sorted before overwriting it.